http/vibe/http/common.d

```diff
@@ -768,6 +769,12 @@ final class Cookie {
 	/// ditto
 	@property bool httpOnly() const { return m_httpOnly; }
 
+	/** Prevent cross-request forgery.
+	*/
+	@property void sameSiteStrict(bool value) { m_sameSiteStrict = value; }
```

Comment:
Since this is the lowest API level, it would be nice to add a `Cookie.SameSite` enum with `default`, `lax` and `strict`, and a `@property void sameSite(SameSite)`.